test/belay_brokerage_test.exs

```diff
@@ -29,12 +29,31 @@ defmodule BelayBrokerageTest do
       assert investor.postal_code == received_investor.postal_code
       assert investor.email == received_investor.email
       assert investor.phone == received_investor.phone
+      assert investor.access_token == nil
+    end
+
+    test "when field changes value, existing investor is updated" do
+      investor = build(:investor)
+
+      assert {:ok, %Investor{} = received_investor} = BelayBrokerage.upsert_investor(@default_tenant, investor)
```

Comment:
And we should clean up the warnings. I think in our other repos we've got warnings compiled as errors but not here
```suggestion
      assert {:ok, %Investor{}} == BelayBrokerage.upsert_investor(@default_tenant, investor)
```